W-12245690 Update reference (#2192)
mulesoft/docs-connectors: db/1.13/modules/ROOT/pages/database-documentation.adoc

This operation does not support streaming, meaning that there is no need to perform additional transformations to the payload in order to access the operation results. This behavior is identical to the *Query Single* operation released in version 1.9.

> Even though, the *Fetch size* field relies on streaming, you can use it for this source. The *Max rows* field values are ignored by the connector.  In the next Database Connector version, the *Fetch size* field will be removed for this source to use the correct *Max rows* field instead.

Reviewer: ```suggestion
Even though the *Fetch size* field relies on streaming, you can use it for this source. The *Max rows* field values are ignored by the connector.  In the next Database Connector version, the *Fetch size* field will be removed for this source to use the correct *Max rows* field instead.
```

This operation does not support streaming, meaning that there is no need to perform additional transformations to the payload in order to access the operation results. This behavior is identical to the *Query Single* operation released in version 1.9.

> Currently, for Database Connector 1.13, even though, the *Fetch size* field relies on streaming, you can use it for this source. Though, the *Max rows* field values are ignored by the connector. In the next Database Connector version, the *Fetch size* field will be removed for this source to use the correct *Max rows* field instead.

Reviewer 2: ```suggestion
For Database Connector 1.13, even though the *Fetch size* field relies on streaming, you can use it for this source because the connector ignores the *Max rows* field values. In the next Database Connector version, the *Fetch size* field will be removed for this source and the source will use the correct *Max rows* field instead.
```
